shell/app/modules/application/common/components/list-input-group.tsx

```diff
@@ -30,8 +31,20 @@ interface IVariableInputGroupProps {
 export default class extends PureComponent<IVariableInputGroupProps, any> {
   constructor(props:IVariableInputGroupProps) {
     super(props);
+    const ids:string[] = [];
+    const _values = map((props.value || []), (item)=> {
+      let id:string = uuid();
+      while(ids.includes(id)) {
```

Comment:
I'm suspecting `ids` should always be empty in this constructor, so does this `while` take any effect?